pkg/scheduler/plugins/predicates/proportional.go

```diff
@@ -27,6 +27,12 @@ import (
 // checkNodeResourceIsProportional checks if a gpu:cpu:memory is Proportional
 func checkNodeResourceIsProportional(task *api.TaskInfo, node *api.NodeInfo, proportional map[v1.ResourceName]baseResource) (*api.Status, error) {
 	status := &api.Status{}
+
+	if task.Preemptable {
```

Comment:
The task here is the role of the preemptor, and `task.Preemptable` indicates whether the current task is allowed to be preempted.
Sorry, I don't quite understand the purpose of the judgment here.

Reply:
Main idea of this PR, that preemptable tasks can skip this proportional check, because they can be preempted, so there are no problem to schedule them. In system which I maintain in my organisation should be available ability to schedule preemptible tasks bypassing proportional plugin.

Reply:
Thanks for your response, I see what you mean.

However, the proportional logic is designed to ensure that resources are available in the allocate scene, and is not bound to preempt. In reality, many users will not configure preempt actions.

I am worried that the modification here will affect the existing proportional function. So I have reservations about the current pr, and I can refer to the ideas of other contributors in the community.

Reply:
Maybe we can add an optional parameter to proportional plugin configuration, which by default will save current behavior but allow to enable changes made by this pr?

Reply:
It sounds more reasonable to add a config. And what's more, job's queue should support `Reclaimable` so that the job      in this queue can be reclaimed.